### PR TITLE
Sanitize CI commit message extraction `Test`

### PR DESCRIPTION
**Pull request by ilyasfoo**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Updates `get_commit_message` script to use environment variables to obtain commit message and PR title.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. E2e CI runs should pass

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [x] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

Update `get_commit_message` CI to use environment variables for commit message and PR title

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/global-setup.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = async ( config ) => {
 	}
 	try {
 		fs.unlinkSync( process.env.CUSTOMERSTATE );
-		console.log( 'Customer state file deleted successfully.' );
+		console.log( 'Customer state file deleted successfully' );
```
Comment (rjchow) [inline plugins/woocommerce/tests/e2e-pw/global-setup.js:34]: is this a lint issue? 👀

**Review by rjchow — APPROVED** [review]
code looks fine to me! thanks for jumping on this

File: plugins/woocommerce/tests/e2e-pw/global-setup.js (hunk 1 above)
Comment (ilyasfoo) [reply plugins/woocommerce/tests/e2e-pw/global-setup.js:34]: It's to trigger test 😬

**Comment by ilyasfoo** [comment]
Thanks, @rjchow! However, this PR is closed in favour of removing BuildKite: https://github.com/woocommerce/woocommerce/pull/47449

**Pull request closed without merging**